chart/epinio-installer/ci/k3d-values.yaml

```diff
@@ -0,0 +1,60 @@
+## Default values for Epinio Helm Chart.
```

Comment:
Is this file a replacement of the default chart  values.yaml because then I would just specify what need to be overridden.
I am mainly concerned by the following line parameters

```
containerRegistryChart: "https://github.com/epinio/epinio-helm-chart/releases/download/container-registry-0.1.0/container-registry-0.1.0.tgz"
epinioChart: "https://github.com/epinio/epinio-helm-chart/releases/download/epinio-0.2.0/epinio-0.2.0.tgz"
traefikChart: "https://helm.traefik.io/traefik/traefik-10.3.4.tgz"
``` 

I am wondering if this file should only contain

```
skipTraefik: true
```

Reply:
Yes, that's just a file to customize the test we are doing on top of k3d. 
https://github.com/helm/charts/blob/master/test/README.md#providing-custom-test-values
It can be useful if we want to test deployments with different parameters.
Maybe it can be improved with template but for now, I don't know, purpose is to have a quick CI to start.

Reply:
It works perfectly with only `skipTraefik: true` in the file, thanks for that good point.